aps/utils/roxar/grid_model.py

```diff
@@ -135,26 +135,39 @@ def average_of_property_inside_zone_region(
     Returns a dictionary with parameter name as key and average as value.
     """
     cell_index_defined = find_defined_cells(
-        zone_values, zone_number, region_values, region_number, debug_level=debug_level
+        zone_values, zone_number, region_values, region_number
     )
+    print('{cell_index_defined=}')
     if len(cell_index_defined) == 0:
-        raise ValueError(
+        print('Error:')
+        print(
             f'Zone number {zone_number} specified in the GUI has no active cells in '
             f'the zone parameter {GridModelConstants.ZONE_NAME}.'
-            'There is probably a mismatch between the zone number in the GUI and zone values '
-            'for the active grid cells in the zone parameter. A possible fix is to delete the '
-            'zone parameter and let APS automatically re-create it. '
-            'This situation can happen if the zone parameter was originally created for a multi-zone grid '
-            'but later split into single zone grids while still keeping the original zone values.'
         )
-    if debug_level >= Debug.VERY_VERBOSE:
-        check_for_nan(cell_index_defined, 'cell_index_defined')
+        print(
+            'There is probably a mismatch between the zone number in the GUI and zone values'
+        )
+        print('for the active grid cells in the zone parameter.')
+        print(
+            'A possible fix is to delete the zone parameter and let APS automatically re-create it.'
+        )
+        print(
+            'This situation can happen if the zone parameter was originally created for '
+        )
+        print(
+            'a multi-zone grid but later split into single zone grids while still keeping'
+        )
+        print('the original zone values.')
```

Comment:
I think you could achieve the same look by doing a single print statement, but with multiple string statements all ending in `\n`.

So something like 

```python
print(
  'The first line. \n'
  'The second line. \n'
)
```

Reply:
My experience with RMS log window is that it tend to ignore the '\n' if I do it in that way. It looks better in RMS log window when running RMS from workflow manager if I have separate print statements.